plugin-flex-ts-template-v2/src/feature-library/chat-transfer/flex-hooks/chat-orchestrator/completed.ts

```diff
@@ -0,0 +1,12 @@
+import * as Flex from "@twilio/flex-ui";
+
+export const chatOrchestratorHook = (flex: typeof Flex, manager: Flex.Manager) => ({
+  event: 'completed',
```

Comment:
Following the conventions in this PR, should the different events for the `chatOrchestrator` also have their own enum to reference the different events? Might be too granular, but thought I'd bring it up

Reply:
I'm only aware of wrapup and completed, will try to find others and define an enum for it.